sponsorship-prospectus.md

```diff
@@ -0,0 +1,186 @@
+pick one - letting you be discovered and known to the cream of
+the community at maybe half (or even less!) of your marketing
+budget.
+
+### Flexibility
+
+Have something specific in mind? Please don’t hesitate to ask!
+We are more than happy to work with you and tailor the
+sponsorship benefits to suit your needs.
+
+
+### Audience profile
+
+PyCon India is attended by a diverse mix of people across
+domains and expertise which includes and not limited to -
+Developers, technologists, programmers, educators, bloggers,
+authors, web developers, CTOs, managers, entrepreneurs,
+scientists, engineers and domain experts.
+
+<br>
+
+## Sponsorship Packages
+
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <td colspan="1"><strong>Package</strong></td>
+      <td colspan="2"><strong>Price</strong></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td><strong>INR</strong></td>
+      <td><strong>USD</strong></td>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Diamond</td>
+      <td>₹ 6,00,000</td>
+      <td>$ 8000</td>
+    </tr>
+    <tr>
+      <td>Platinum</td>
+      <td>₹ 4,50,000</td>
+      <td>$ 6000</td>
+    </tr>
+    <tr>
+      <td>Gold</td>
+      <td>₹ 3,00,000</td>
+      <td>$ 4000</td>
+    </tr>
+    <tr>
+      <td>Silver</td>
+      <td>₹ 1,50,000</td>
+      <td>$ 2000</td>
+    </tr>
+    <tr>
+      <td>Startup</td>
+      <td>₹ 50,000</td>
+      <td>$ 700</td>
+    </tr>
+  </tbody>
+</table>
+
+The above mentioned sponsorship amount is exclusive of 18% GST.
+
+<br>
+
+## Sponsorship Benefits
+
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <td><strong>Benefit</strong></td>
+      <td><strong>Diamond</strong></td>
+      <td><strong>Platinum</strong></td>
+      <td><strong>Gold</strong></td>
+      <td><strong>Silver</strong></td>
+      <td><strong>Startup</strong></td>
+    </tr>
+  </thead>
+  <tbody>
+    {% for benefit in site.data.sponsorship.benefits %}
+      <tr>
+        <td>{{benefit.type}}</td>
+        <td>{{benefit.diamond | sponsor_benefit}}</td>
+        <td>{{benefit.platinum | sponsor_benefit}}</td>
+        <td>{{benefit.gold | sponsor_benefit}}</td>
+        <td>{{benefit.silver | sponsor_benefit}}</td>
+        <td>{{benefit.Startup | sponsor_benefit}}</td>
+      </tr>
+    {% endfor %}
+  </tbody>
+</table>
+
+
+<br>
+
+## Privacy Policy
+
+Participant details will not be shared without their explicit permission.
+Participants are allowed to share their information with you at your stall.
+Give them a good reason to do so. Make a compelling pitch for yourself 
+or offer goodies in exchange for contact information.
+
+<br>
+
+## Contact
+
+- Mail: [sponsorship@in.pycon.org](mailto:sponsorship@in.pycon.org)
+- *Abhishek Mishra*, Sponsorship Coordinator
+  - Mobile: +91-70664 87364
+- *Chandan Kumar*, Sponsorship Coordinator
+  - Mobile: +91- 96072 64082
+- *Sayan Chowdhury*, PyCon India 2020 Chair
+  - Mobile: +91-96869 92532
```

Comment:
Are you sure with putting out the phone numbers in public?

Reply:
Yes, good point, removing that.